### DISCUSSION TITLE
UI (different screens with transparency and uneven buttons in the new version)

### DISCUSSION THREAD

**Discussion by PatronNick** [lobehub/sd-webui-lobe-theme#595] [General]
![Untitled](https://github.com/user-attachments/assets/d917dabf-1795-4f7a-910d-f48ea50cc28f)

**Comment by jipsita** [comment]
Make an "user.css" file in root directory and paste this inside and save:

.forge-image-container {
    /* Sets the dark base color */
    background-color: rgb(17, 17, 17) !important;

    /* This is the complex gradient that creates the checkered pattern */
    background-image: linear-gradient(45deg, rgba(255, 255, 255, 0.06) 25%, transparent 25%), linear-gradient(-45deg, rgba(255, 255, 255, 0.06) 25%, transparent 25%), linear-gradient(45deg, transparent 75%, rgba(255, 255, 255, 0.06) 75%), linear-gradient(-45deg, transparent 75%, rgba(255, 255, 255, 0.06) 75%) !important;
    
    /* Defines the size of the checkered pattern squares */
    background-size: 20px 20px !important;
    
    /* Positions the gradients to form the pattern correctly */
    background-position: 0px 0px, 0px 10px, 10px -10px, -10px 0px !important;
}
